tests/WarpTest/WarpTest.fsproj

```diff
@@ -262,7 +262,7 @@
     <When Condition="($(TargetFrameworkIdentifier) == '.NETCoreApp' And $(TargetFrameworkVersion) == 'v5.0') Or ($(TargetFrameworkIdentifier) == '.NETStandard' And $(TargetFrameworkVersion) == 'v2.0')">
       <ItemGroup>
         <Reference Include="FSharp.Core">
-          <HintPath>C:\Users\andra\.nuget\packages\fsharp.core\5.0.0\lib\netstandard2.0\FSharp.Core.dll</HintPath>
+          <HintPath>..\..\..\..\..\..\.nuget\packages\fsharp.core\5.0.0\lib\netstandard2.0\FSharp.Core.dll</HintPath>

```

Comment:
Note that both the original and this change seems bogus.

Reply:
WarpTest project is not live, just kept for eventual revival, this is no matter.